template/deny.toml

```diff
@@ -7,20 +8,13 @@ targets = [
 ]
 
 [advisories]
-vulnerability = "warn"
-unmaintained = "allow"
```

Comment:
note: Just making sure this is deliberate. Removing `unmaintained` here will now default to `deny`. Is this the new intended behaviour?

Reply:
Yes. It is. According to [cargo-deny](https://github.com/EmbarkStudios/cargo-deny/releases/tag/0.16.0) at least.

We'll have to manually list all _unmaintained_ advisories in the _ignore_ section now.
I tried it with the hdfs operator and didn't get any warnings but you're correct: I should announce when this happens and I'll be there for the cleanup.